python/tvm/meta_schedule/tune.py

```diff
@@ -108,7 +109,7 @@ def tune_tasks(
     elif not isinstance(database, Database):
         database = Database.create(database, module_equality=module_equality)
     if not isinstance(cost_model, CostModel):
-        cost_model = CostModel.create(cost_model, num_tuning_cores=num_cores)
+        cost_model = CostModel.create(cost_model, num_tuning_cores=num_cores, **kwargs)
```

Comment:
Same as previous comment, IMHO if we only want to add a single parameter to the cost model, it might be more specific to directly create a cost model with `tree_method` parameter or you can modify the default construction parameters.